README.md

```diff
@@ -1,33 +1,29 @@
+NGINX Service Mesh is a fully integrated lightweight service mesh that leverages a data plane powered by NGINX Plus to manage container traffic in Kubernetes environments.
 
-### Resources
+This repository contains the implementation for the `nginx-meshctl` command-line tool, as well as helm charts and example configurations.
+
+## Resources
 
 - [Product Details](https://www.nginx.com/products/nginx-service-mesh/)
 - [NGINX Service Mesh Documentation](https://docs.nginx.com/nginx-service-mesh)
 - [NGINX Plus Kubernetes Ingress Controller](https://www.nginx.com/products/nginx-ingress-controller/)
 - [NGINX Plus Product Details](https://www.nginx.com/products/nginx/)
 
-### Commercial Support
-
-For paid customers of NGINX Service Mesh, commercial support is available at the [NGINX Support](https://www.nginx.com/support/) website.
-
-### Git Issues
+## Getting Started
 
-For NGINX Service Mesh support or issues not addressed by documentation, you can reach out via the [Issues tab](https://github.com/nginxinc/nginx-service-mesh/issues).
+Follow our [Getting Started](https://docs.nginx.com/nginx-service-mesh/get-started/) guides to get NGINX Service Mesh up and running.
 
-We use issues for bug reports and to discuss new features. Creating issues is good, creating good issues is even better. Filing meaningful bug reports with lots of information in them helps us figure out what to fix when and how it impacts our users. We like bugs because it means people are using our code, and we like fixing them even more. 
+## Contacts
 
-All issues should follow these guidelines:
+We’d like to hear your feedback! If you have any suggestions or experience issues with our service mesh, please create an issue or send a pull request on GitHub.
+You can contact us directly via [kubernetes@nginx.com](mailto:kubernetes@nginx.com) or on the [NGINX Community Slack](https://nginxcommunity.slack.com/channels/nginx-service-mesh).
 
-- Describe the problem. Include version of Kubernetes, `nginx-meshctl version`, and what Kubernetes platform.
-- Include detailed information about how to recreate the issue.
-- Include relevant configurations, error messages, and logs.
-- Sanitize the data. For example, be mindful of IPs, ports, application names, and URLs.
+## Contributing
 
-### NGINX Plus Kubernetes Ingress Controller Support
+If you'd like to contribute to the project, please read our [Contributing guide](CONTRIBUTING.md).
 
-If you are using NGINX Service Mesh with NGINX Plus Ingress Controller for Kubernetes, you can get support through your usual channels.
+## Support
 
-Existing NGINX and F5 customers can reach out to their account team(s) for help and support with NGINX Service Mesh.
+For information on support, view our [Support](https://docs.nginx.com/nginx-service-mesh/support/contact-support/) doc.
```

Comment:
I would add a `SECURITY.md` like https://github.com/nginxinc/kubernetes-ingress/blob/main/SECURITY.md

Reply:
It already exists in the repo, do you want me to link to it in the README?

Reply:
Oh I didn't see that it was already in the repo. Yeah I think it makes sense to link it here since the link for the doc is part of that already, but up to you 🙂